### ISSUE TITLE
Centralized feed syncing.

### ISSUE THREAD

**Issue by kickscondor** [kickscondor/fraidycat#5]
I think it's going to become helpful to offer a centralized sync option. This would make a mobile version possible. (Ideally you could keep mobile, Chrome and Beaker all synced.)

This could be:

* Dat. If I can figure out a way to connect to the Dat from a web extension.
* Micropub. This way you can publish sync data as a blogroll on your site.
* [Firefox Sync API](https://wiki.mozilla.org/CloudServices/Sync). If it's possible to access extension sync data from the API, then I'm happy with offering this to people who trust Mozilla.
* Dropbox / Google Drive. Last resort.

**Comment by kickscondor** [comment]
Since I'm already doing this as a browser extension, it makes sense to support Firefox and Chrome APIs, if possible. (In some ways, it seems like Fraidycat is an adapter between personal storage and personal network. Many possible outputs supported, many possible inputs.)

After doing some studying, it seems that Firefox Accounts API may be capable of this.

Signing-in to the account through the API:
https://mozilla.github.io/application-services/docs/accounts/welcome.html

APIs available after login:
https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Atttached_APIs

But, really, the key discover is this PDF - it's dated Oct 23, 2019 - is it accurate??
https://buildmedia.readthedocs.org/media/pdf/mozilla-services/latest/mozilla-services.pdf

**Comment by andrew-wja** [comment]
Hi there,

I'm logged in to two firefox instances (one on windows, one on linux, on the same machine). According to the project landing page, it already supports sync if you're logged in to your browser "And, if you are logged-in to your browser, Fraidycat will sync your follows between your different computers."

However Fraidycat is empty on my windows Firefox instance! Other extensions are syncing, so I don't think it's an issue with Firefox sync specifically. Is this really supported right now? I didn't want to create a new issue after seeing this one, so I thought I'd ask first 😅

**Comment by kickscondor** [comment]
Yes - this definitely works. Although Firefox sometimes needs to run for a few minutes before it picks up the sync. Let me do some testing and get back to you.

**Comment by justinkelly** [comment]
@kickscondor thanks for your amazing work on friadycay - re this issue 
* are the 2 firefox supposed to be online at the same time for sync to work (or at least to first iniate)?

I have the same issue firefox on win 10 and firefox on linux - not syncing at the moment - and the machines/instances are never online at the same time

**Comment by kickscondor** [comment]
Absolutely fine if they aren't online simultaneously - the data is shared in Firefox's "Kinto" storage, which is an online database. Clearly there is a bug here. Thank you for confirmation, Justin.

**Comment by andrew-wja** [comment]
I came back to say that this suddenly started working for me yesterday! I opened fraidycat in Firefox on my macbook and lo and behold, all my feeds started to sync where previously they had just been empty! 

I know this is supremely unhelpful, but it works now :D

**Comment by medicinecal** [comment]
Glad to hear it. However the current sync setup is not ideal. For this reason, we're moving to a proper database in Fraidycat 2. The extension will continue to receive bugfixes, but we'll be encouraging people to move to Fraidycat 2.

**Comment by joevo2** [comment]
Does chrome actually sync the feed? I saw that it should sync in the chrome extension descriptions but doesnt seems to actually sync or syncing is a FireFox only thing for now?

**Comment by kickscondor** [comment]
Yes - I use Chrome's sync. However, it's kind of clunky - in that it can take time for your follows to appear.

So you're signed in to Chrome on both computers? Both have the Fraidycat extension? One of them shows all of your follows and one is empty?

**Comment by joevo2** [comment]
it does work but it takes quite awhile like you mentioned. great works! btw any work being done on the mobile app version or is it still in the pipeline?

**Comment by kickscondor** [comment]
I am collaborating with someone on another project at the moment and will return to Fraidycat in March. One of the top priorities at that time will be mobile support. Thanks for your interest! We'll see how it goes...
